dlink-admin/src/main/java/com/dlink/dto/StudioExecuteDTO.java

```diff
@@ -31,6 +31,7 @@
 
 import lombok.Getter;
 import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
 
```

Comment:
Please use `import org.slf4j.Logger` instead of `lombok.extern.slf4j.Slf4j`.

Reply:
this is used for @Slf4j on class, so not need to write `private static final Logger log = LoggerFactory.getLogger(StudioExecuteDTO.class);`,  it's fact used the `org.slf4j.Logger` interface.

Reply:
> this is used for @slf4j on class, so not need to write `private static final Logger log = LoggerFactory.getLogger(StudioExecuteDTO.class);`, it's fact used the `org.slf4j.Logger` interface.

Yes, your method code is more concise.